### ISSUE TITLE
Implement ToQueryString()

### ISSUE THREAD

**Issue by lauxjpn** [PomeloFoundation/Pomelo.EntityFrameworkCore.MySql#999]
See https://github.com/aspnet/EntityFrameworkCore/pull/19368#event-2905414096 for further details.

**Comment by neonVoice** [comment]
@lauxjpn do you think, its possible for me to implement this based on pull request you provided? I'd like to try. If yes - is it better to make a new branch from main and assign pull request to this issue somehow ? Not quite sure how to start working on another issue, if all pushes to my fork are associated with another issue already.

**Comment by neonVoice** [comment]
From what i understood, `EF.Relational` changes are not yet incorporated in Pomelo repository, so its currently not possible to implement `MySqlServerQueryStringFactory` ? Or this factories/interfaces should be placed somewhere else?

**Comment by lauxjpn** [comment]
I haven't taken a look at the implementation details of this yet, but it is EF Core 5 related. I can't give you a comprehensive tutorial about how to work with git and GitHub here, but there are many tutorials out there.

Concretely in this case, you would create a local branch to work on the issue (and not work directly on the `master` branch as you did previously). Then you push your branch to your fork and create a PR when you are done. Since this issue is EF Core 5 related, it needs to be merged with the `5.0-wip` branch instead of `master`.

**Comment by neonVoice** [comment]
@lauxjpn yeah, thanks, i've spent some time reading and watching some basic stuff, those questions are now completely irrelevant (forgot to remove). Anyway, i read through #19368 commit and there are some changes that should be incorporated first, for example `IRelationalQueryStringFactory`. To be more precise, every change that likes in `EF.Relational` folder and which is not present in Pomelo repository, even though `Microsoft.EntityFrameworkCore.Query.Internal` namespace is used there.

**Comment by lauxjpn** [comment]
I see. I have planed to get the provider on par with the current development state of EF Core in May. There are still more pressing matters for our `3.2.0` releases though, which will consume my attention before that.

May is probably a good time to start doing the work on this, because its about half a year away from the .NET 5 GA release, so major features will be relatively stable and we can provide valuable feedback to the EF Core team.
